Makefile

```diff
@@ -1,28 +1,42 @@
 BIN = node_modules/.bin
 DUO = $(BIN)/duo
-MINIFY = $(BIN)/uglifyjs
-WATCH = $(BIN)/wr
-DELEGATE = test test-browser test-sauce test-coverage
+T = $(BIN)/duo-test -m test/api.js -R spec -P 8378
 
-BUILD = ./build
-WATCH_FILES = lib index.js component.json Makefile
+SRC = index.js $(shell find lib -type f -name '*.js')
+TESTS = $(wildcard test/*.test.js)
 
-build: node_modules
-	@mkdir -p $(BUILD)
-	@$(DUO) --quiet --stdout --global recurly index.js > $(BUILD)/recurly.js
-	@$(MINIFY) $(BUILD)/recurly.js --output $(BUILD)/recurly.min.js
+test: test-phantomjs
+
+test-phantomjs: build/recurly.min.js build/test.js
+	@$(T) phantomjs
+
+test-browser: build/recurly.min.js build/test.js
+	@$(T) browser
+
+test-sauce: BROWSER ?= ie:9
```

Comment:
default sauce test browser is ie9. It is typically the most problematic; thus, running `make test-sauce` will let you run the suite on ie9 and interact with the browser remotely during the suite run.